base/ltclass.dtx

```diff
@@ -904,6 +907,7 @@
 \def\IfFormatAtLeastTF{\@ifl@t@r\fmtversion}
 \let\IfPackageAtLeastTF\@ifpackagelater
 \let\IfClassAtLeastTF\@ifclasslater
+\long\def\IfFileAtLeastTF#1{\expandafter\@ifl@t@r\csname ver@#1\endcsname}
```

Comment:
I don't think it is helpful to make everything long when a \par really only means that there is some error

Reply:
I've adjusted (though I guess I'm with Hans - I'd simply simply ignore the difference entirely)

Reply:
It gives different error messages and I think it is better to have "paragraph ended before ..." instead of missing \endcsname inserted"